### PR TITLE
Add workflow howto

### PR DESCRIPTION
**Pull request by tatyree**
Ticket: https://waffle.io/ministryofjustice/cloud-platform/cards/5afaad17e9ca30001c31cb42

This is a basic HOWTO walking through the steps needed to raise new tickets/issues, how to get them through the workflow, and a few best practices for working with tickets.

### CODE REVIEW

File: docs/HOWTO-workflow.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+   2) `Delete Branch`
+10) Move your ticket to done.
+
+## Getting a quick estimate online
+
+If you added an unesitmated ticket to `To Do`, or you need to get an
+estimate on a ticket do this:
+
+1) Copy the ticket link into `#cloud-platform` slack
+2) Copy this code into slack in a link immediately following the link: `/poll “Estimate” “1"“2”“3"“5”“8"`
```
Comment (tariqmoj) [inline docs/HOWTO-workflow.md]: `/poll “Estimate <INSERT TICKET URL>” “1"“2”“3"“5”“8"`

Comment (kalbir) [reply docs/HOWTO-workflow.md]: I'd like people to paste the *title* of the ticket into the poll as well please. It gives you a lot more context than the URL.

File: docs/HOWTO-workflow.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+6) Open the `ministryofjustice/cloud-platform` project board.
+7) Click on the blue `Add Issue` button at the top of the page.
+8) Add a brief, descriptive title.
+5) Add some details about your issue.
+9) Pick the repo where the issue/ticket should be created--if you aren't
+sure which repo, ask the Product or Delivery Manager.
+10) Select the column where the ticket should start:
+    * `Inbox` is for as-yet-unestimated featrue work and should be **mostly** in priority order;
+    * `Support To Do` for incoming support requests;
+    * `To Do` for items to go into the current sprint or in one of the
```
Comment (kalbir) [inline docs/HOWTO-workflow.md]: I'd make this just the current sprint. We should be working through the inbox to move things  into the To Do list regularly enough that we don't miss things.

File: docs/HOWTO-workflow.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+      next two or three sprints.  These should be kept in priority order.
+11) Associate the ticket with a label or milestone as required (both
+are optional.
+12) Save.
+
+## Adding a ticket from a repo associated with waffle.io
+
+1) Go to the repo.
```
Comment (kalbir) [inline docs/HOWTO-workflow.md]: A little bit more descriptive maybe? e.g. "Go to the repo that your issue relates to (if you can't work out which one go to the `cloud platform` repo).

File: docs/HOWTO-workflow.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+## Working on a ticket
+
+1) Click on the person icon in the top-right-hand corner of the ticket.
+2) Assign yourself.
+3) Move the ticket to `To Do`.
+4) Do the thing!
+5) Create a Pull Request.
+   * The ticket should have the main details of the job.
+   * The Pull Request should contain an executive summary of this in its
+    description.
```
Comment (kalbir) [inline docs/HOWTO-workflow.md]: Maybe link to this: https://www.annashipman.co.uk/jfdi/good-pull-requests.html

**Pull request merged**